src/commentbot/CommentBot.jl

```diff
@@ -210,8 +210,8 @@ function action(rp::RequestParams{T}, zsock::RequestSocket) where T <: RegisterT
             set_error_status(rp)
         end
     catch ex
-        bt = get_backtrace(ex)
-        @info("Unexpected error: $bt")
+        @info "Unexpected error" exception = (ex, catch_backtrace())
+        bt = sprint(Base.showerror, ex, catch_backtrace())
```

Comment:
This introduces a tiny bit of duplication in two places. But I feel that if `raise_issue` should be doing some special printing, then there should probably be a special method for that function.